Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.messages.in

```diff
@@ -38,6 +38,7 @@ messages -> ModelProcessModelPlayerProxy {
     SetPaused(bool paused) -> (bool succeeded) Async
     SetCurrentTime(Seconds currentTime) -> () Async
     SetEnvironmentMap(Ref<WebCore::SharedBuffer> data)
+    SetHasPortal(bool hasPortal)
```

Comment:
This should be marked with `[EnabledBy=NonPortaledModelEnabled]`

Reply:
Using the runtime check in HTMLModelElement as @rniwa suggested 🙂

Reply:
Verified locally with `nonPortaledModelEnabled` on `WKPreferences` turned off, `noportal` attribute doesn't work; and when it's on, `noportal` takes effect.